manifests/init.pp

```diff
@@ -114,6 +115,7 @@
   Optional[Hash]                      $ci_unicorn                      = undef,
   Boolean                             $config_manage                   = true,
   Stdlib::Absolutepath                $config_file                     = '/etc/gitlab/gitlab.rb',
+  Boolean                             $config_show_diff                = false,
```

Comment:
As far as I can tell, the previous code did show the diff so if we want to be backward compatible, this should default to true?

Also, when we switch to epp templates, this setting would be useless if I recall correctly because in this case the diff is not shown when it contains sensitive data.

Reply:
> As far as I can tell, the previous code did show the diff so if we want to be backward compatible, this should default to true?

That is also what I thought previously, and was then asked to put a secure default value.  But any value is OK for me, i’ll set the one you want!

> 
> Also, when we switch to epp templates, this setting would be useless if I recall correctly because in this case the diff is not shown when it contains sensitive data.

As said in the PR description, I first tried to convert to an `epp` template without success. As it was more than 3 month ago, I do not remember why exactly…  But you are right, once the template is converted to `epp`, the strings marked as `Sensitive()` will be obfuscated.